pulp_smash/tests/ostree/api_v2/test_copy.py

```diff
@@ -0,0 +1,73 @@
+# coding=utf-8
+"""Copy content between OSTree repositories with a filter."""
+import unittest
+from urllib.parse import urljoin
+
+from pulp_smash import api, config, utils
+from pulp_smash.constants import OSTREE_BRANCHES, OSTREE_FEED, REPOSITORY_PATH
+from pulp_smash.tests.ostree.utils import gen_distributor, gen_repo
+from pulp_smash.tests.ostree.utils import set_up_module as setUpModule  # noqa pylint:disable=unused-import
+
+
+class CopyTestCase(unittest.TestCase):
+    """Copy content between OSTree repositories with a filter."""
+
+    def test_all(self):
+        """Copy content between OSTree repositories with a filter..
+
+        Verify that:
+
+        Filters can be used to restrict which units
+        are copied between repositories.
```

Comment:
Nitpick: Make this "Verify that filters can be used..." statement a single paragraph, instead of splitting it across two paragraphs.